proposals/2021/Add MultiTenant.md

```diff
@@ -0,0 +1,54 @@
+# Add MultiTenant to Building
+
+## Overview
+
+This proposal is to add the `MultiTenant` element as a child element of `Building` element, with boolean options.  
+
+## Justification
+
+True if the building is a multi-tenant building.
+
+## UDFs
+
+Currently this is conveyed in Audit Template via:
+`/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:UserDefinedFields/auc:UserDefinedField[auc:FieldName/text() = Multi Tenant]/auc:FieldValue`.
+Our proposal is to add it under a `Building`.
+
+## Example
+
+```xml
+    <auc:Facilities>
+        <auc:Facility ID="F1">
+            <auc:Buildings>
+                <auc:Building ID="B1">
+                    <auc:MultiTenant>True</auc:MultiTenant>
+                </auc:Building>
+            </auc:Buildings>
+        </auc:Facility>
+    </auc:Facilities>
+```
+
+## Implementation
+
+```xml
+      <xs:element name="Buildings" minOccurs="0">
+        <xs:complexType>
+          <xs:sequence>
+            <xs:element name="Building" type="auc:BuildingType" maxOccurs="unbounded">
+              <xs:annotation>
+                <xs:documentation>A building is a single structure wholly or partially enclosed within exterior walls, or within exterior and abutment walls (party walls), and a roof, affording shelter to persons, animals, or property. A building can be two or more units held in the condominium form of ownership that are governed by the same board of managers.</xs:documentation>
```

Comment:
This sounds like a copy/paste from some other document. Should we reference the document here?

Reply:
> This sounds like a copy/paste from some other document. Should we reference the document here?

It's directly copied from the schema. I'm kind of just quoting the original schema to indicate which element the change is under.

Reply:
Oh, gotcha.